packages/cli/src/utils.ts

```diff
@@ -222,7 +222,7 @@ export async function readProjectSecrets(
 }
 
 export function writeSecretToFile(secrets: dotenv.DotenvParseOutput, rootFolder: string): void {
-  const secretFile = `${rootFolder}/.${ConfigFolderName}/${getActiveEnv()}.userdata`;
+  const secretFile = `${rootFolder}/.${ConfigFolderName}/${getActiveEnv(rootFolder)}.userdata`;
```

Comment:
do we need to check if `getActiveEnv(rootFolder)` return undefined?

Reply:
Added error handling